test/blackbox-tests/test-cases/dune-project-meta/run.t

```diff
@@ -140,3 +139,33 @@ after running `dune subst`:
   $ grep ^version version/_build/default/META.foo
   version = "1.0"
 
+Generation of opam files with lang dune >= 1.11
+-----------------------------------------------
+
+  $ mkdir gen-v1.11
+  $ cat > gen-v1.11/dune-project <<EOF
+  > (lang dune 1.11)
+  > (name test)
+  > (generate_opam_files true)
+  > (package (name test))
+  > EOF
+
+  $ dune build @install --root gen-v1.11
+  Entering directory 'gen-v1.11'
+  $ cat gen-v1.11/test.opam
+  # This file is generated by dune, edit dune-project instead
+  opam-version: "2.0"
+  build: [
+    ["dune" "subst"] {pinned}
+    [
```

Comment:
This formatting looks a bit goofy to me. Probably doesn't matter, but if it's possible to improve that would be very welcome.

Reply:
I had a look at the opam file printer but it doesn't seem straightforward to do unfortunately :(